ci-operator/step-registry/interop-tests/ocs-tests/interop-tests-ocs-tests-commands.sh

```diff
@@ -10,7 +10,9 @@ OCP_VERSION="${OCP_MAJOR_MINOR}"
 
 OCS_VERSION=$(oc get csv -n openshift-storage -o json | jq -r '.items[] | select(.metadata.name | startswith("ocs-operator")).spec.version' | cut -d. -f1,2)
 
-CLUSTER_NAME=$(cat "${SHARED_DIR}/CLUSTER_NAME")
+# CLUSTER_NAME=$(cat "${SHARED_DIR}/CLUSTER_NAME")
+# CLUSTER_NAME="cluster-name"
+CLUSTER_NAME=$([[ -f "${SHARED_DIR}/CLUSTER_NAME" ]] && cat "${SHARED_DIR}/CLUSTER_NAME" || echo "cluster-name")
```

Comment:
After thinking about it more, we can just pull the cluster name from the cluster itself using :

```sh
oc cluster-info | grep -oP 'https:\/\/api.\K[^.]+'
```

Reply:
Thank you @rabin-io for your help.
Actually  I have already made the previous change and it seems working. The target is to merge this PR asap so that ODF could run in OPP. 
Could you please help to approve it for now so that it could be merged, and feel free to create a PR that includes all the enhancements needed